ethereum/ts-scripts/relayer/config/mainnet/chains.json

```diff
@@ -0,0 +1,76 @@
+{
+  "guardianSetIndex": 3,
+  "description": "This file contains the chains against which all the scripts should run.",
+  "chains": [
+    {
+      "description": "Ethereum",
+      "evmNetworkId": 1,
+      "chainId": 2,
+      "rpc": "https://rpc.ankr.com/eth",
+      "wormholeAddress": "0x98f3c9e6E3fAce36bAAd05FE09d375Ef1464288B"
```

Comment:
why duplicate these from the sdk consts file?

Reply:
This is to keep consistency with how the rest of the configs are done. I think ideally these should not be duplicated here though, and should be taken from the SDK - but probably that should be done in a seperate PR along with all of the other files in relayer/config?